client/app/components/elements/AddNewListingButton/AddNewListingButton.css

```diff
@@ -28,6 +28,7 @@
   left: 0;
   border-radius: calc(var(--AddNewListingButton_height) / 2);
   transition: filter 100ms ease-in;
+  transition: background-color 100ms ease-in;
```

Comment:
To my understanding, this will now override the filter transition. Multiple transitions need to be comma-separated, i.e. `transition: filter 100ms ease-in, background-color 100ms ease-in`. Or is this different in CSS Next?